.travis.yml

```
@@ -97,22 +70,47 @@ notifications:
 
 jobs:
   include:
+    - stage: Test
+      name: "Node 0"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=0
+    - stage: Test
+      name: "Node 1"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=1
+    - stage: Test
+      name: "Node 2"
+      env: KNAPSACK_PRO_CI_NODE_INDEX=2
+    - stage: Test
+      name: "Preact, Storybook, bundle audit, console check, e2e"
```

nitpick (non-blocking): You don't need to mention Preact. Maybe instead put front-end tests.

---

```suggestion
      name: "Front-end tests, Storybook, bundle audit, console check, e2e"
```
:+1: lets change it.